src/js/modules/RoutingApp.js

```diff
@@ -1,25 +1,28 @@
 import Provider from 'react-redux/lib/components/Provider';
 import React from 'react';
-import {Route, Switch} from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
 import appHistory from 'tools/appHistory';
 import ConnectedRouter from 'react-router-redux/ConnectedRouter';
 import store from '../store';
 
 import core from './core';
 import articles from './articles';
 import sections from './sections';
+import users from './users';
 
-const {HomePage, PageLayout} = core.components;
-const {ArticlePage} = articles.components;
-const {SectionPage} = sections.components;
+const { HomePage, PageLayout } = core.components;
+const { ArticlePage } = articles.components;
+const { SectionPage } = sections.components;
+const { RolePage, UserPage } = users.components;
 
 // TODO: change to mapStateToProps
 const allSectionRoutes = sections.selectors.getAllSectionRoutes(store.getState());
+const allRoles = users.selectors.getAllRoles(store.getState());

```

Comment:
If I connect() a mapStateToProps, I get `Invariant Violation: Could not find "store" in either the context or props of "Connect(RoutingApp)". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(RoutingApp)".`